docs/api-docs/cart-and-checkout/embedded-checkout-tutorial.md

```diff
@@ -150,7 +161,28 @@ Next, generate a cart URL and set this cart as the active cart by posting to  to
   "embedded_checkout_url": "https://store-id30h7ohwf.mybigcommerce.com/cart.php?embedded=1&action=loadInCheckout&id=bc218c65-7a32-4ab7-8082-68730c074d02&token=aa958e2b7922035bf3339215d95d145ebd9193deb36ae847caa780aa2e003e4b"
 }
 ```
-<a id="step-4-embed-checkout"></a>
+
+### Redirecting A Logged-In Customer to Embedded Checkout
+For some use cases, you may want your your customer to be logged in before they can begin the checkout process.
+
+The can be done using the [Customer Login API](https://developer.bigcommerce.com/api-docs/customers/customer-login-api#logging-in-a-customer). 
+
+Your app will need to use JSON Web Token Standard to create a new token. Use a [JWT library](https://jwt.io/#libraries-io) to accomplish this. Include the `checkout_url` as part of the request payload you send to BigCommerce:
```

Comment:
Can also link to the relevant functions in our API clients - we have one in PHP, Ruby, and Python.